scripts/release.sh

```diff
@@ -50,9 +50,9 @@ function tag_images_in_yamls() {
   for file in $@; do
     [[ "${file##*.}" != "yaml" ]] && continue
     echo "Inspecting ${file}"
-    for image in $(grep -o "${DOCKER_BASE}/[a-z\./-]\+@sha256:[0-9a-f]\+" ${file}); do
+    for image in $(grep -o "${DOCKER_BASE}/[a-z\./-]\+@sha256:[0-9a-f]\+" "${file}"); do
       for region in "" ${GEO_REGIONS// /. }; do
-        gcloud -q container images add-tag ${image} ${region}${image%%@*}:${TAG}
+        gcloud -q container images add-tag "${image}" "${region}""${image%%@*}":"${TAG}"
```

Comment:
The last part would be better as `"${region}${image%%@*}:${TAG}"` without all the extra double-quotes inside

Reply:
done